atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/transaction/impl/SnapshotTransaction.java

```diff
@@ -1684,9 +1705,11 @@ private boolean rollbackOtherTransaction(long startTs, TransactionService transa
             transactionService.putUnlessExists(startTs, TransactionConstants.FAILED_COMMIT_TS);
             return true;
         } catch (KeyAlreadyExistsException e) {
-            String msg = "Two transactions tried to roll back someone else's request with start: " + startTs;
-            log.error("This isn't a bug but it should be very infrequent. {}", msg,
-                    new TransactionFailedRetriableException(msg, e));
+            log.error("This isn't a bug but it should be very infrequent. Two transactions tried to roll back someone"
```

Comment:
It shouldn't be an error. We have warn log tracking in many places now, so would pick up if this was frequent.

Reply:
Yea I'd mark this INFO at highest